testfiles/HospitalBaseline.idf

```diff
@@ -57774,8 +57774,8 @@
     AHU2_OAMinOAFracSchedule,!- Minimum Fraction of Outdoor Air Schedule Name
     ,                        !- Maximum Fraction of Outdoor Air Schedule Name
     ,                        !- Mechanical Ventilation Controller Name
-    No,                      !- Time of Day Economizer Control Schedule Name
-    ,                        !- High Humidity Control
+    ,                        !- Time of Day Economizer Control Schedule Name
+    No,                      !- High Humidity Control
```

Comment:
@Myoldmopar Yes, these are intentional.  These files were wrong from day 1 way back when, but there was never any getinput code to validate these.  And since I reverted the economizer changes back out of this, there still isn't any.  I could at least add those input checks back into this PR if you like.